common/laws/00_monarchy.txt

```diff
@@ -62,13 +62,12 @@ monarchy_military_reforms = {
 	
 	relaxed_recruiting_standards = { #Noble Retinues - Macedon
 		modifier = {
-			levy_size_multiplier = 0.05
-			happiness_for_same_culture_modifier = 0.05
-			happiness_for_wrong_culture_modifier = -0.05
+			levy_size_multiplier = 0.025
+			global_population_happiness = 0.05
 			discipline = 0.025
 			land_morale_modifier = 0.025
-			experience_decay = -0.1
-			monthly_military_experience = 0.1
+			experience_decay = -0.2
```

Comment:
less experience a month, but better levy xp retention that can be converted to xp when disbanding.